purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition.py

```diff
@@ -77,8 +77,6 @@ def _get_requisition_line_search_domain(self, requisition, item):
                  item.product_uom_id.id or False),
                 ('account_analytic_id', '=',
                  item.line_id.analytic_account_id.id or False)]
-        if not item.product_id:
-            vals['name'] = item.name
```

Comment:
You can do the changes in your PR.

On Mon, Feb 15, 2016 at 9:33 AM, Kitti U. <notifications@github.com> wrote:

> In
> purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition.py
> <https://github.com/Eficent/purchase-workflow/pull/2#discussion_r52870962>
> :
>
> > @@ -77,8 +77,6 @@ def _get_requisition_line_search_domain(self, requisition, item):
> >                   item.product_uom_id.id or False),
> >                  ('account_analytic_id', '=',
> >                   item.line_id.analytic_account_id.id or False)]
> > -        if not item.product_id:
> > -            vals['name'] = item.name
>
> @jbeficent <https://github.com/jbeficent>
> Will you want me to make changes in my PR, or you will just correct on
> your branch and just ignore this? Which is better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Eficent/purchase-workflow/pull/2/files#r52870962>.
>



-- 
Jordi Ballester Alomar
Founder | Eficent
(+34) 629530707 | jordi.ballester@eficent.com | http://www.eficent.com
Twitter: https://twitter.com/jbeficent_erp | Linkedin:
https://www.linkedin.com/in/jordiballesteralomar